project1/styles.css

```
@@ -0,0 +1,60 @@
+h1 {
+    font-size: 70px;
+    text-align: center;
+    background-color: darkgrey;
+    font-family: 'IM Fell French Canon SC', serif;
+    color: #034247;
+
+}
+
+h2 {
+    font-size: 20px;
+    text-align: center;
+    background-color: darkgrey;
+}
+
+body {
+    margin: 0 auto;
+    max-width: 800px;
+    background-color: darkgrey;
+    font-family: 'Abel', sans-serif;
+    color: black;
+}
+
+hr {
+    width: 100%;
+    height: 2px;
+    background: #fff;
+}
+
+div {
+    text-align: center;
+    font-size: 80px;
+}
+
+p {
+    font-size: 20px;
+}
+
+.sub-border {
+    background-color: cadetblue;
+    text-align: left;
+    padding: 50px;
+}
+
+blockquote {
+    font-size: 45px;
+    text-align: center;
+    color: #034247;
+    font-family: 'IM Fell French Canon SC', serif;
+}
+
+.star {
+    color: white;
+    font-family: 'Notable', sans-serif;
+    font-size: 30px;
+}
```

See note above. If you delete the other two text alignment rules, you'll need to add one here
```suggestion
    text-align: center;
}
```